### PR TITLE
tsp, bump cadl-ranch

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**

### CODE REVIEW

File: typespec-extension/src/code-model-builder.ts [hunk 1]
```diff
@@ -305,6 +305,8 @@ export class CodeModelBuilder {
             extensions: {
               "x-ms-skip-url-encoding": schema instanceof UriSchema,
             },
+            // // make the logic same as TCGC, which takes the service default of host as client default
+            // clientDefaultValue: getDefaultValue(it.defaultValue),
```
Comment (weidongxu-microsoft) [inline typespec-extension/src/code-model-builder.ts:309]: currently disable this line, not sure the status of corresponding TCGC behavior https://github.com/Azure/typespec-azure/issues/1030

File: typespec-extension/package.json [hunk 2]
```diff
@@ -61,7 +61,6 @@
     "lodash": "~4.17.21"
   },
   "devDependencies": {
-    "@azure-tools/typespec-autorest": "0.43.0",
```
Comment (weidongxu-microsoft) [inline typespec-extension/package.json:64]: It is removed from `@azure-tools/typespec-azure-resource-manager`

File: typespec-tests/Generate.ps1 [hunk 3]
```diff
@@ -61,6 +61,13 @@ $generateScript = {
     $tspOptions += " --option ""@azure-tools/typespec-java.stream-style-serialization=true"""
     # for mgmt, do not generate tests due to random mock values
     $tspOptions += " --option ""@azure-tools/typespec-java.generate-tests=false"""
+    # also generate with group-etag-headers=false since mgmt doesn't support etag grouping yet
+    $tspOptions += " --option ""@azure-tools/typespec-java.group-etag-headers=false"""
+  } elseif ($tspFile -match "azure[\\/]arm[\\/]") {
+    # for mgmt, do not generate tests due to random mock values
+    $tspOptions += " --option ""@azure-tools/typespec-java.generate-tests=false"""
```
Comment (weidongxu-microsoft) [inline typespec-tests/Generate.ps1:68]: disable mock test

**Review by XiaofeiCao — APPROVED** [review]

File: typespec-tests/Generate.ps1 (hunk 3 above)
Comment (weidongxu-microsoft) [reply typespec-tests/Generate.ps1]: struggling on the regex, going to take a while

**Pull request merged**